_posts/wfh1972/2016-01-18-wfh1972-turtlemagic.md

```diff
@@ -0,0 +1,8 @@
+--- 
+layout: post
+author: wfh1972 (Wayne Homan)
```

Comment:
I'm going to go ahead and merge this but the (Wayne Homan) will confuse Jekyll.  While helpful to a human, Jekyll doesn't recognize `wfh1972 (Wayne Homan)`